library/src/main/java/com/google/android/exoplayer2/extractor/flv/FlvExtractor.java

```diff
@@ -51,6 +52,7 @@
   private static final int STATE_SKIPPING_TO_TAG_HEADER = 2;
   private static final int STATE_READING_TAG_HEADER = 3;
   private static final int STATE_READING_TAG_DATA = 4;
+  private static final int STATE_SEEK_FLV = 5;
```

Comment:
Are these FLV changes (the ones to this file and to ScriptTagPayloadReader) actually related to adding RTMP support? If not they should be in a separate pull request.